pkg/distributor/query.go

```diff
@@ -53,6 +54,32 @@ func (d *Distributor) Query(ctx context.Context, from, to model.Time, matchers .
 	return matrix, err
 }
 
+func (d *Distributor) QueryExemplars(ctx context.Context, from, to model.Time, matchers ...[]*labels.Matcher) (*ingester_client.QueryResponse, error) {
+	var result *ingester_client.QueryResponse
+	err := instrument.CollectedRequest(ctx, "Distributor.QueryExemplars", d.queryDuration, instrument.ErrorCode, func(ctx context.Context) error {
```

Comment:
As a followup to this it may be worth updating the help text for queryDuration. Currently it's:

```
Time spent executing expression queries.
```

It may be worth changing it to

```
Time spent executing expression and exemplar queries.
```